src/domains/Auth/api/api.ts

```diff
@@ -102,11 +97,27 @@ export async function verifyEmail(userId: string) {
     return axios.post('/api/confirm/user-email', { userId });
 }
 
+export async function logout() {
+    return axios.post('/api/users/logout');
+}
+
 export async function getMyInfo() {
     // gets user info from the token in header
     return axios.get<ClientSafeUser>('/api/users/me');
 }
 
-export async function logout() {
-    return axios.post('/api/users/logout');
+export async function changeName(fname: string, lname: string) {
+    return axios.put<{ user: ClientSafeUser }>(`/api/users/me/name/`, { fname, lname });
```

Comment:
the trailing / is unnecessary -- style consistency

Reply:
Done